### ISSUE TITLE
Compress web assets related to event tile backgrounds

### ISSUE THREAD

**Issue by autumnfound** [EclipseFdn/events.eclipse.org#99]
Web assets added for new react events tiles were added in an uncompressed state. This could lead to slowdowns for some users on limited connections. We should compress images to be closer to 50kb (where possible while maintaining clarity)

**Comment by chrisguindon** [comment]
@autumnfound Are you sure we need to optimize the images? I took a quick look and they are close to 50kb!

https://github.com/EclipseFdn/events.eclipse.org/blob/master/less/images/TrainingSeries.png

**Comment by autumnfound** [comment]
Apologies, it looks like that is indeed the case! The images I was seeing are the ones that are committed in the static/images folder like the following:

https://github.com/EclipseFdn/events.eclipse.org/blob/master/static/images/EclipseCon.jpg
https://github.com/EclipseFdn/events.eclipse.org/blob/master/static/images/unknown.jpg
https://github.com/EclipseFdn/events.eclipse.org/blob/master/static/images/otherInteresting.jpg

Additionally, 3.38mb image:
https://github.com/EclipseFdn/events.eclipse.org/blob/master/static/images/Jumbotron-bg.png

We should maybe rephrase this bug to clean up the images folder which is incredibly cluttered by no longer used assets (anything to do with the gmap implementation).

**Comment by autumnfound** [comment]
Also, we currently load an image that's a black square. Is there any reason we can't replace that with a black background?

**Comment by flora8984461** [comment]
> Also, we currently load an image that's a black square. Is there any reason we can't replace that with a black background?

Good point, I thought there would be image for unknown type, my last pr just did the default black color, I will make a new pr to remove that black image and just use black color, thanks

**Comment by chrisguindon** [comment]
`Additionally, 3.38mb image:
https://github.com/EclipseFdn/events.eclipse.org/blob/master/static/images/Jumbotron-bg.png`

We definitely need to shrink this one :) Good catch!

**Comment by flora8984461** [comment]
I think that link is previously generated by webpack mix() ? Since this is the image I uploaded and I already compressed to 80kb. Should I delete the one in static folder and re-run the webpack? 🤔

https://github.com/EclipseFdn/events.eclipse.org/blob/master/less/images/Jumbotron-bg.jpg

**Comment by chrisguindon** [comment]
No, we are good! The image on prod is around 90kb!
